PROFILING.md

```diff
@@ -0,0 +1,46 @@
+h1. Profiling linkerd-tcp
```

Comment:
Hmm, I still see this `h1.` header in the latest commit. Maybe your change didn't get pushed?